src/libraries/System.Net.Security/tests/FunctionalTests/MD4Tests.cs

```diff
@@ -0,0 +1,97 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.IO.Tests;
+using System.Net.Security;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Security.Tests
+{
+    public class MD4Tests
+    {
+        // MD4 ("") = 31d6cfe0d16ae931b73c59d7e0c089c0
+        [Fact]
+        public void RFC1320_Test1() 
+        {
+            MD4 hash = new MD4();
+            byte[] input = new byte[0];
+            byte[] expected = { 0x31, 0xd6, 0xcf, 0xe0, 0xd1, 0x6a, 0xe9, 0x31, 0xb7, 0x3c, 0x59, 0xd7, 0xe0, 0xc0, 0x89, 0xc0 };
+            Verify(hash, input, expected);
+        }
+
+        private void Verify(MD4 hash, byte[] input, byte[] expected) 
+        {
+            // Span<byte> outputSpan = stackalloc byte[expected.Length];
```

Comment:
Can this and the commented out `Assert.Equal` be removed?

Reply:
Did the `Span<byte>` not work? It looks like the expected type of MD4 HashData is a Span.

Reply:
removed